internal/lockclient/client.go

```diff
@@ -4,7 +4,18 @@ import "github.com/SystemBuilders/LocKey/internal/lockservice"
 
 // Client describes a client that can be used to interact with
 // the Lockey lockservice. The client can start the lockservice
-// and interact acquire and release locks by making calls to it.
+// and interact by making calls to it.
+//
+// The client has the ability to start the lockservice from its
+// in-built function or it can be started separately.
+//
+// The client offers the user to Acquire a lock, Release a lock,
```

Comment:
does ' the client offers the functional of Acquiring and Releasing an object using its descriptor' sound better?

Reply:
Ill go with `the client allows the user`.